### PR TITLE
Translate Ruby 3.4.0 released (de)

### PR DESCRIPTION
**Pull request by nethad**
Same as last year, I've translated the release announcement into German. Let me know if you have any comments :+1:

### CODE REVIEW

**Comment by nethad** [comment]
@hsbt you merged my PR last year, maybe you're the right person to review? Thank you :)

File: de/news/_posts/2024-12-25-ruby-3-4-0-released.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+- String-Literale in Dateien ohne einen `frozen_string_literal`-Kommentar geben jetzt eine Deprecation-Warnung aus, wenn sie mutiert werden. Diese Warnungen können mit `-W:deprecated` oder durch Setzen von `Warning[:deprecated] = true` aktiviert werden. Um diese Änderung zu deaktivieren, können Sie Ruby mit dem Kommandozeilenargument `--disable-frozen-string-literal` ausführen. [[Feature #20205]]
+
+- Keyword-Splatting von `nil` beim Aufrufen von Methoden wird jetzt unterstützt. `**nil` wird ähnlich wie `**{}` behandelt, übergibt keine Schlüsselwörter und ruft keine Konvertierungsmethoden auf. [[Bug #20064]]
+
+- Die Blockübergabe ist im Index nicht mehr erlaubt. [[Bug #19918]]
+
+- Keyword-Argumente sind im Index nicht mehr erlaubt. [[Bug #20218]]
+
+- Der Toplevel-Name `::Ruby` ist jetzt reserviert, und die Definition wird vewarnt, wenn `Warning[:deprecated]` aktiviert ist. [[Feature #20884]]
```
Comment (bovi) [inline de/news/_posts/2024-12-25-ruby-3-4-0-released.md:102]: ```vewarnt```

File: de/news/_posts/2024-12-25-ruby-3-4-0-released.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+  - `Ractor.main?` wurde hinzugefügt. [[Feature #20627]]
+
+  - `Ractor.[]` und `Ractor.[]=` wurden hinzugefügt, um auf den ractor-lokalen Speicher des aktuellen Ractors zuzugreifen. [[Feature #20715]]
+
+  - `Ractor.store_if_absent(key){ init }` wurde hinzugefügt, um ractor-lokale Variablen threadsicher zu initialisieren. [[Feature #20875]]
+
+- Range
+
+  - `Range#size` löst jetzt `TypeError` aus, wenn der Bereich nicht iterierbar ist. [[Misc #18984]]
```
Comment (bovi) [inline de/news/_posts/2024-12-25-ruby-3-4-0-released.md:132]: Range#size löst jetzt ```einen``` TypeError aus

File: de/news/_posts/2024-12-25-ruby-3-4-0-released.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+          from test.rb:2:in `<main>'
+
+  Neu:
+  test.rb:1:in 'Object#foo': undefined method 'time' for an instance of Integer
+          from test.rb:2:in '<main>'
+  ```
+
+- Die Darstellung von Hash#inspect wurde geändert. [[Bug #20433]]
+
+  - Symbolschlüssel werden im modernen Symbolschlüssel-Syntax angezeigt: `"{user: 1}"`
```
Comment (bovi) [inline de/news/_posts/2024-12-25-ruby-3-4-0-released.md:183]: Symbolschlüssel werden ```in der``` modernen Symbolschlüssel-Syntax angezeigt

File: de/news/_posts/2024-12-25-ruby-3-4-0-released.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+Siehe [NEWS](https://docs.ruby-lang.org/en/3.4/NEWS_md.html)
+oder [Commit-Logs](https://github.com/ruby/ruby/compare/v3_3_0...{{ release.tag }})
+für weitere Details.
+
+Mit diesen Änderungen wurden [{{ release.stats.files_changed }} Dateien geändert, {{ release.stats.insertions }} Einfügungen(+), {{ release.stats.deletions }} Löschungen(-)](https://github.com/ruby/ruby/compare/v3_3_0...{{ release.tag }}#file_bucket)
+seit Ruby 3.3.0!
+
+Frohe Weihnachten, schöne Feiertage und viel Spaß beim Programmieren mit Ruby 3.4!
+
+## Download
```
Comment (bovi) [inline de/news/_posts/2024-12-25-ruby-3-4-0-released.md:258]: Herunterladen

File: de/news/_posts/2024-12-25-ruby-3-4-0-released.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+      SHA512: {{ release.sha512.xz }}
+
+- <{{ release.url.zip }}>
+
+      SIZE: {{ release.size.zip }}
+      SHA1: {{ release.sha1.zip }}
+      SHA256: {{ release.sha256.zip }}
+      SHA512: {{ release.sha512.zip }}
+
+## What is Ruby
```
Comment (bovi) [inline de/news/_posts/2024-12-25-ruby-3-4-0-released.md:281]: Was ist Ruby?

**Review by bovi — COMMENTED** [review]
Looks good to me. Here some remarks from my side.

**Comment by nethad** [comment]
@bovi Thanks for the review, I fixed all the comments

**Comment by bovi** [comment]
@nethad thanks

**Pull request merged**